content/tutorials/manage-the-rippled-server/capacity-planning.md

```diff
@@ -57,7 +57,7 @@ path={path_to_ledger_store}
 
 ### Historical Data
 
-The amount of historical data that a `rippled` server keeps online is a major contributor to required storage space. At the time of writing (2018-03-01), a `rippled` server stores about 12GB of data per day. You can expect this amount to grow as transaction volume increases across the XRP Ledger network. You can control how much data you keep with the `online_delete` and `advisory_delete` fields.
+The amount of historical data that a `rippled` server keeps online is a major contributor to required storage space. At the time of writing (2018-05-30), a `rippled` server stores about 12GB of data per day and requires 7.9TB to store the full history of the XRP Ledger. You can expect this amount to grow as transaction volume increases across the XRP Ledger network. You can control how much data you keep with the `online_delete` and `advisory_delete` fields.
```

Comment:
Per the JIRA ticket, it's up to 8.4 TB as of 2018-10-29

Reply:
Thank you for keeping an eye out! Yes, saw the comment from Mark 👍 , just hadn't had time to update the PR. Updated now! 🎊